cmd/cephcsi.go

```diff
@@ -129,7 +129,7 @@ func init() {
 	flag.BoolVar(&conf.EnableProfiling, "enableprofiling", false, "enable go profiling")
 
 	// CSI-Addons configuration
-	flag.StringVar(&conf.CSIAddonsEndpoint, "csi-addons-endpoint", "unix://tmp/csi-addons.sock", "CSI-Addons endpoint")
+	flag.StringVar(&conf.CSIAddonsEndpoint, "csi-addons-endpoint", "unix:///csi/csi-addons.sock", "CSI-Addons endpoint")
```

Comment:
not related to this PR but looks like this new parameter missing in doc https://github.com/ceph/ceph-csi/blob/devel/docs/deploy-rbd.md#configuration

Reply:
right! I'll add it there  too

Reply:
this has been addressed too.